src/components/page-components/Home.jsx

```diff
@@ -1,11 +1,78 @@
+import { useEffect, useState } from 'react';
 import ArticleContainer from '../home-components/ArticleContainer';
 import Sort from '../home-components/Sort';
+import { api } from '../../api';
+import { useSearchParams } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
 
 const Home = () => {
+	const navigate = useNavigate();
+	const [articles, setArticles] = useState([]);
+	const [isLoading, setIsLoading] = useState(true);
+	const [searchParams, setSearchParams] = useSearchParams();
+
+	const sortArticles = ({ sortValue, orderValue }) => {
```

Comment:
We shouldn't be sorting our articles using JS methods- this should be done by our back-end using the query string that we provide. You will need to refactor your useEffect to depend on the selected order and sort state you have in place, and then refetch the data whenever a new query is needed.